### PR TITLE
Task/dtserwfour 561 single app program email and program selection view allow landscape mode

### PR DESCRIPTION
**Pull request by shanavas123**

### CODE REVIEW

**Review by SaiNarendran — APPROVED** [review]

File: receiptui/src/main/java/com/hyperwallet/android/ui/receipt/viewmodel/TabbedListReceiptsViewModel.java [hunk 1]
```diff
@@ -48,6 +48,7 @@
     private MutableLiveData<Boolean> retryListReceipts = new MutableLiveData<Boolean>();
     public ProgramModel mProgramModel;
     private boolean mIsInitialized;
+    private boolean mIsPortraitMode;
```
Comment (rbanikPP) [inline receiptui/src/main/java/com/hyperwallet/android/ui/receipt/viewmodel/TabbedListReceiptsViewModel.java:51]: Why is this variable introduced? should not it be always portrait?

Comment (shanavas123) [reply receiptui/src/main/java/com/hyperwallet/android/ui/receipt/viewmodel/TabbedListReceiptsViewModel.java:51]: UI-sdk, common sdk, it has the option to set portrait/landscape Hyperwallet app to set as value as true to lock portrait only.

**Comment by shanavas123** [comment]
portrait/landscape mode verified in demo-app, UI-SDK working fine in both orientation.

File: transferui/src/main/java/com/hyperwallet/android/ui/transfer/view/ListTransferSourceFragment.java [hunk 2]
```diff
@@ -61,7 +61,7 @@
     private RecyclerView mRecyclerView;
     private ListTransferSourceAdapter mListTransferSourceAdapter;
     private String mActiveTransferSourceToken;
-    private List<TransferSource> mTransferSourceList = new ArrayList<>();
+    private ArrayList<TransferSource> mTransferSourceList = new ArrayList<>();
```
[the remark this answers is not in the document]
Comment (shanavas123) [reply transferui/src/main/java/com/hyperwallet/android/ui/transfer/view/ListTransferSourceFragment.java:64]: if any orientation changes to maintain data, we store and retrieve data from saved instance, saved instance have option to store parcableArraylist.

[the remark this answers is not in the document]
Comment (shanavas123) [reply transferui/src/main/java/com/hyperwallet/android/ui/transfer/view/ListTransferSourceFragment.java]: fixed

**Review by rbanikPP — APPROVED** [review]

**Review by vwagh-hw — APPROVED** [review]

**Pull request merged**